kore/src/Kore/Strategies/Goal.hs

```diff
@@ -879,11 +894,9 @@ derivePar rules goal = errorBracket $ do
     configuration :: Pattern Variable
     configuration = getConfiguration goal
 
-    errorBracket action =
-        onException action
-            (formatExceptionInfo
-                ("configuration=" <> unparseToText configuration)
-            )
+    enrichEventualException action =
```

Comment:
Please lift this to the top level as:

```.hs
mapGoalException :: MonadCatch m => Goal goal => goal -> m a -> m a
```

Reply:
Actually, I think I would prefer naming the type `WithConfiguration` and the function `withConfiguration`. The name `GoalException` sounds like there is a problem with the `Goal`, which is not exactly what's happening.